### PR TITLE
Use the default UI font on the welcome screen

### PR DESCRIPTION
**Pull request by Mailaender**
instead of Arial which is a proprietary font by @microsoft.

### CODE REVIEW

File: chemclipse/plugins/org.eclipse.chemclipse.ux.extension.ui/src/org/eclipse/chemclipse/ux/extension/ui/swt/TaskTile.java [hunk 1]
```diff
@@ -254,16 +255,21 @@ private void updateStyle(int style) {
...
-			fontSize = 18;
 		}
-		//
-		textSection.setFont(Fonts.getCachedFont(getDisplay(), "Arial", fontSize, SWT.BOLD)); //$NON-NLS-1$
+		FontData[] fontDatas = label.getFont().getFontData();
+		for(FontData fontData : fontDatas) {
+			fontData.setHeight(fontSize);
+			fontData.setStyle(SWT.BOLD);
+		}
+		textSection.setFont(new Font(getDisplay(), fontDatas));
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.ux.extension.ui/src/org/eclipse/chemclipse/ux/extension/ui/swt/TaskTile.java]: Does the font need to be disposed?
https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fswt%2Fgraphics%2FFont.html

Comment (Mailaender) [reply chemclipse/plugins/org.eclipse.chemclipse.ux.extension.ui/src/org/eclipse/chemclipse/ux/extension/ui/swt/TaskTile.java]: I think you are right however we never dispose the welcome view.

**Comment by eselmeister** [comment]
Why do you create the `FontRegistry`, instead of using the existing approach via `Fonts`?
E.g. to avoid the Windows font name?
`Fonts.getCachedFont(getDisplay(), label.getFont().getFontData()[0].getName(), fontSize, SWT.BOLD)`

**Comment by Mailaender** [comment]
I don't think that the font size is already cached and I don't see where that is disposing fonts.

**Comment by eselmeister** [comment]
`Fonts` takes care of caching and disposing if needed. Do you dispose it via `FontRegistry`?

**Comment by Mailaender** [comment]
https://github.com/eclipse/chemclipse/blob/3eaa42707b827c81e1081782cb457a6248929b0d/chemclipse/plugins/org.eclipse.chemclipse.swt.ui/src/org/eclipse/chemclipse/swt/ui/support/Fonts.java#L26 are never disposed.

[FontRegistry](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjface%2Fresource%2FFontRegistry.html) will automatically dispose when the display is disposed so we don't have to.

**Comment by eselmeister** [comment]
I don't like to introduce two concepts with this approach. I would then recommend to use the `FontRegistry` inside of `Fonts`, so that developer can easily rely on the Fonts class instead of creating a new `FontRegistry` whenever they get in touch with using fonts.

**Pull request merged**